Sources_v3/Database/DTOs/ChannelDTO.swift

```diff
@@ -25,6 +25,7 @@ class ChannelDTO: NSManagedObject {
     @NSManaged var createdBy: UserDTO
     @NSManaged var team: TeamDTO?
     @NSManaged var members: Set<MemberDTO>
+    @NSManaged var typingMembers: Set<MemberDTO>
```

Comment:
should we maybe call it `currentlyTypingMembers` or `membersTyping`? @b-onc

Reply:
fixed in 5f783af14c08fc2741c818742753c34dd920f5a3